### PR TITLE
Fix "A PART OF" Code style for non-public, non-static fields #8387

### PR DESCRIPTION
**Pull request by Arnold2381**
## Purpose / Description
Change the formatting issues according to #8370 

## Fixes
Fixes some of #8387 

## Approach
Followed #8370, added those changes ran gradlew lintRelease to generate a report of all the issues and solve errors accordingly. 

## Checklist
_Please, go through these checks before submitting the PR._

- [x] You have not changed whitespace unnecessarily (it makes diffs hard to read)
- [x] You have a descriptive commit message with a short title (first line, max 50 chars).
- [x] Your code follows the style of the project (e.g. never omit braces in `if` statements) 
- [x] You have commented on your code, particularly in hard-to-understand areas
- [x] You have performed a self-review of your own code
- [x] UI changes: include screenshots of all affected screens (in particular showing any new or changed strings)
- [x] UI Changes: You have tested your change using the [Google Accessibility Scanner](https://play.google.com/store/apps/details?id=com.google.android.apps.accessibility.auditor)

### CODE REVIEW

File: AnkiDroid/src/main/java/com/ichi2/ui/ButtonItemAdapter.java [hunk 1]
```diff
@@ -43,21 +43,21 @@ of this software and associated documentation files (the "Software"), to deal
  */
 public class ButtonItemAdapter extends RecyclerView.Adapter<ButtonItemAdapter.ButtonVH> {
 
-    private final ArrayList<String> items;
-    private ItemCallback itemCallback;
-    private ButtonCallback buttonCallback;
+    private final ArrayList<String> mItems;
```
Comment (david-allison-1) [inline AnkiDroid/src/main/java/com/ichi2/ui/ButtonItemAdapter.java:46]: This one looks strange in the GitHub viewer (single color) - seems correct

**Review by david-allison-1 — APPROVED** [review]
Looks good to me assuming CI passes

I expect this will cut down most of the warnings

**Comment by Arnold2381** [comment]
> Looks good to me assuming CI passes
> 
> I expect this will cut down most of the warnings

once it gets merged I will pull upstream and run the lint check and see the remaining warnings.

File: AnkiDroid/src/main/java/com/ichi2/ui/ButtonItemAdapter.java (hunk 1 above)
Comment (mikehardy) [reply AnkiDroid/src/main/java/com/ichi2/ui/ButtonItemAdapter.java:46]: LGTM :shrug:

**Review by mikehardy — APPROVED** [review]
LGTM - thank you!

**Pull request merged**